platform/sun50i/prcm.c

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright © 2017-2018 The Crust Firmware Authors.
+ * SPDX-License-Identifier: (BSD-3-Clause OR GPL-2.0)
+ */
+
+#include <dm.h>
+#include <error.h>
+#include <mmio.h>
+#include <prcm.h>
+#include <platform/devices.h>
+
+#define VDD_SYS_PWROFF_GATING (DEV_R_PRCM + 0x0110)
+
+int
+prcm_shutdown(void)
+{
+	/* Validate power off gates for shutdown. */
```

Comment:
What do you mean by "validate"?

Reply:
This was the terminology used in the manual, but it's not very clear. I will update this.

Reply:
Changed to "activate power off gating". Thoughts?